app/routes/signup.tsx

```diff
@@ -135,61 +144,209 @@ export const loader: LoaderFunction = async ({request}) => {
+        team.focusClassName,
+        {
+          'ring-2 ring-offset-team-current': selected,
+          'dark:ring-offset-gray-900 ring-offset-white': !selected,
+        },
+      )}
+    >
+      {selected ? (
+        <span className="absolute left-9 top-9 text-team-current">
+          <CheckIcon />
+        </span>
+      ) : null}
+
+      <label className="block pb-12 pt-20 px-12 text-center">
+        <input
+          className="sr-only"
+          type="radio"
+          name="team"
+          value={value}
+          aria-describedby={error ? 'team-error' : undefined}
+        />
+        <img className="block mb-16" src={team.image.src} alt="" />
+        <H6>{team.label}</H6>
+      </label>
+    </div>
+  )
+}
+
 export default function NewAccount() {
   const data = useRouteData<LoaderData>()
+  const [formValues, setFormValues] = React.useState({
+    firstName: '',
+    team: '',
+  })
+
+  const formIsValid =
+    formValues.firstName.trim().length > 0 &&
+    Object.keys(TEAM_MAP).includes(formValues.team)
+
   return (
-    <div>
-      <h2>Welcome to kentcdodds.com</h2>
-      <p>To create your account for {data.email}, choose your team:</p>
-      <Form method="post" noValidate>
-        <div>
-          <label htmlFor="firstName">First Name: </label>
-          <input
-            name="firstName"
-            id="firstName"
-            required
-            defaultValue={data.fields?.firstName ?? ''}
-          />
-          {data.errors?.firstName ? (
-            <div
-              role="alert"
-              className="dark:text-red-300 text-red-800"
-              id="body-error"
-            >
-              {data.errors.firstName}
+    <div className="mt-12">
+      <style>{`
+          :root {
+            --color-team-current: var(--color-team-${(
+              formValues.team || 'UNKNOWN'
+            ).toLowerCase()}); 
+          }
+        `}</style>
+      <Form
+        className="mb-64"
+        method="post"
+        onChange={event => {
+          const form = event.currentTarget
+          setFormValues({
+            firstName: form.firstName.value,
+            team: form.team.value,
+          })
+        }}
+      >
+        <Grid>
+          <div className="col-span-full mb-12 lg:mb-20">
+            <div className="flex items-center justify-between">
+              <div>
+                <H2 className="mb-2">Let’s start with choosing a team.</H2>
+                <H2 variant="secondary" as="p">
+                  You can’t change this later.
+                </H2>
+              </div>
+              <Link
+                to="/logout"
+                className="flex col-span-full items-center px-11 py-6 text-black dark:text-white border-2 border-gray-600 rounded-full space-x-4 lg:col-span-8 lg:col-start-3"
+              >
+                <LogoutIcon />
+                <H6 as="span">logout</H6>
+              </Link>
```

Comment:
They're not actually logged in until they've created an account so I'm not sure it makes sense to include the logout button here 🤔

Reply:
Happy to remove it. Just say the word. 😅

Reply:
Yeah, let's do. We can use it on the profile page though